package.json

```diff
@@ -24,7 +24,7 @@
 		"@nextcloud/dialogs": "^3.2.0",
 		"@nextcloud/event-bus": "^3.0.1",
 		"@nextcloud/initial-state": "^2.0.0",
-		"@nextcloud/l10n": "^2.0.0",
+		"@nextcloud/l10n": "^2.0.1",
```

Comment:
There is a "bug" I added in `2.0.1`. In English there are tons of warnings in the console with "No translations found" because in English we use keys as the translations. It impacts performance on development with opened console end decrease DX. I want to fix it today.